tests/conformance/README.md

```diff
@@ -119,4 +119,63 @@ If you want to combine VS Code & dlv for debugging so you can set breakpoints in
+
+1. In the `conformance.yml` you should create a new step in a workflow for your component that creates new env variables. You will need a variable for each specific resource your tests will use. If you require 3 different topics and 2 different tables for your tests you should have 5 different env variables set. The only convention you must follow for the variables is the value must use `env.UNIQUE_ID` to ensure there are no conflicts with the resource names.
+
+```bash
+PUBSUB_AWS_SNSSQS_QUEUE="testQueue-${{ env.UNIQUE_ID }}"
+echo "PUBSUB_AWS_SNSSQS_QUEUE=$PUBSUB_AWS_SNSSQS_QUEUE" >> $GITHUB_ENV
+```
+
+2. When updating the `tests.yml` defined inside `tests/config/<COMPONENT-TYPE>/` folder you should overwrite the default names of any resources the conformance tests use. These values should reference env variables which should be defined in the conformance.yml.
+
+```yaml
+- component: aws.snssqs.terraform
+    operations: ["publish", "subscribe", "multiplehandlers"]
+    config:
+    pubsubName: aws-snssqs
+    testTopicName: ${{PUBSUB_AWS_SNSSQS_TOPIC}}
+    testMultiTopic1Name: ${{PUBSUB_AWS_SNSSQS_TOPIC_MULTI_1}}
+    testMultiTopic2Name: ${{PUBSUB_AWS_SNSSQS_TOPIC_MULTI_2}}
+```
+
+3. When writing your `component.yml` you should reference your credentials using env variables and any resources specified in the yaml should use env variables as well just as you did in the `test.yml`. Also
+`disableEntityManagement` should be set to `true` since we want to use only terraform to provision resources.
+
+```yaml
+metadata:
+    - name: accessKey
+    value: ${{AWS_ACCESS_KEY_ID}}
+    - name: secretKey
+    value: ${{AWS_SECRET_ACCESS_KEY}}
+    - name: region
+    value: "us-east-1"
+    - name: consumerID
+    value: ${{PUBSUB_AWS_SNSSQS_QUEUE}}
+    - name: disableEntityManagement
+    value: "true"
+```
+
+
+4. You will need to create a new terrafrorm file `component.tf` to provision your resources. The file should be placed in its own folder in the `.github/infrastructure/terraform/conformance` directory such as 
+`.github/infrastructure/terraform/conformance/pubsub/aws/snsqsq`. The terraform file should use a UNIQUE_ID variables and use this variables when naming its resources so they matched the names defined earlier. Make sure any resources your tests will use are defined in terraform.
+
+```
+variable "UNIQUE_ID" {
+    type        = string
+    description = "Unique Id of the github worklow run."
+}
+```
+
+5. The component should be added to the `cron-components` step in conformance test workflow `.github/conformance.yml`. The component should have a variable named `terraform-dir` and the value should be the relative path from `.github/infrastructure/terraform/conformance` to the folder which the tests personal terraform files are located such as `pubsub/aws/snsqsq`.
+
+```
+- component: pubsub.aws.snssqs.terraform      
+  terraform-dir: pubsub/aws/snssqs    
 ```
```

Comment:
Can you please create a separate section in the README for AWS specifically that explains what needs to be done when a new AWS conformance or certification test is being added? Isn't there a service principal / service account that needs to be updated? Who can do this and how?